.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,13 @@
 ## Tracking issue
-_https://github.com/flyteorg/flyte/issues/<number>_
-
-<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+<!--
+If your PR fixes an open issue, use `Closes flyteorg/flyte#999` to link your PR with the issue.
+Example: Closes flyteorg/flyte#999
```

Comment:
I think the `owner/repo` keyword is not needed when the PR and Issue are in the same repo

Reply:
Ah yes. I updated the PR to remove the `owner/repo`.